docker-compose.yml

```diff
@@ -375,6 +390,30 @@ services:
       - JSONRPC_HTTP_URL=http://cdk-validium-mock-l1-network:8545
       - DAC_URL=http://cdk-validium-data-availability:8444
       - DAC_ADDRESS=0x70997970c51812dc3a010c7d01b50e0d17dc79c8
+    command:
+      - "/bin/sh"
+      - "-c"
+      - "npm run setup:committee"
+
+  dac-setup-committee-sepolia:
+    container_name: dac-setup-committee
+    image: snapchain/sepolia-cdk-validium-contracts
+    depends_on:
+      cdk-validium-deploy-sepolia:
+        condition: service_completed_successfully
+      cdk-validium-data-availability:
+        condition: service_started
+    volumes:
+      - deploy-output:/shared
+    environment:
+      - JSONRPC_HTTP_URL=https://sepolia.infura.io/v3/<deployer-infura-project-id>
+      - DAC_URL=http://cdk-validium-data-availability:8444
+      - DAC_ADDRESS=0x70997970c51812dc3a010c7d01b50e0d17dc79c8
+      - DEPLOYER_PRIVATE_KEY=<deployer-private-key>
```

Comment:
does this have to be the first key derived from the MNEMONIC above in the other container?

Reply:
yes, from the `MNEMONIC` in `cdk-validium-deploy-sepolia`